packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/java/integrationsteps.md

```diff
@@ -1,3 +1,72 @@
 ## Integration steps
 
-The Java SDK integration steps for this use case are coming soon. See the [Okta Identity Engine Java SDK usage guide](https://github.com/okta/okta-idx-java#usage-guide) for more information.
+### Step 1: User goes to sign-in page
+
+Build a sign-in page for your app that captures both the username and password.
+
+<div class="common-image-format">
+
+![Displays the Java SDK sign-in page](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in-java.png)
+
+</div>
+
+### Step 2: User enters credentials
+
+Begin the authentication process by calling Java SDK's [`IDXAuthenticationWrapper.begin()`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java#L603) method and getting a new [`ProceedContext`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/ProceedContext.java). After the user submits their credentials, call `IDXAuthenticationWrapper.authenticate()` with the credential values.
```

Comment:
> Begin the authentication process by calling **the** Java SDK
[avoid using possessives on objects]

also - what is a new [`ProceedContext`]? Is it a file or a handler or something else?

Reply:
object